static/app/views/dashboards/widgets/timeSeriesWidget/releaseBubbles/createReleaseBubbleHighlighter.tsx

```diff
@@ -1,10 +1,27 @@
 import type {ElementEvent} from 'echarts';
 import type {EChartsInstance} from 'echarts-for-react';
+import debounce from 'lodash/debounce';
 
 import type {Series} from 'sentry/types/echarts';
+import {trackAnalytics} from 'sentry/utils/analytics';
 import {BUBBLE_SERIES_ID} from 'sentry/views/dashboards/widgets/timeSeriesWidget/releaseBubbles/constants';
 import type {Bucket} from 'sentry/views/dashboards/widgets/timeSeriesWidget/releaseBubbles/types';
 
+interface LegendSelectChangedParams {
+  name: string;
+  selected: Record<string, boolean>;
+}
+
+// This needs to be debounced because some charts (e.g. in TimeseriesWidgets)
+// are in a group and share events. Thus on a page with 4 widgets, clicking on
+// a legend item would result in 4 events.
+const trackLegend = debounce((params: LegendSelectChangedParams) => {
+  trackAnalytics('releases.bubbles_legend', {
```

Comment:
I think we're meant to type these events inside of [static/app/utils/analytics/releasesAnalyticsEvents.tsx](https://github.com/getsentry/sentry/blob/cea5fb60ac9f1ed080177c8e01e751f9d1ebdde6/static/app/utils/analytics/releasesAnalyticsEvents.tsx#L25)

there's a `type` and a `const` to update.